src/main/kotlin/ru/buhinder/alcopartyservice/service/EventService.kt

```diff
@@ -118,10 +129,18 @@ class EventService(
             }
     }
 
-    fun findAllByAlcoholicId(alcoholicId: UUID): Mono<List<EventResponse>> {
-        return eventDaoFacade.findAllByAlcoholicIdAndIsNotBanned(alcoholicId)
-            .map { conversionService.convert(it, EventResponse::class.java)!! }
-            .collectList()
+    fun findAllByAlcoholicId(alcoholicId: UUID, page: Int, pageSize: Int): Mono<EventPageableResponse> {
```

Comment:
> return eventDaoFacade.findAllByAlcoholicIdAndIsNotBanned(alcoholicId, page, pageSize)
            .map { conversionService.convert(it, EventResponse::class.java)!! }
            .collectList()
            .zipWith(eventDaoFacade.countAllByAlcoholicIdAndIsNotBanned(alcoholicId))
            .map { EventPageableResponse(it.t1, util.createPagination(it.t2, page, pageSize)) }